.github/workflows/pd-store-ci.yml

```
@@ -153,10 +162,15 @@ jobs:
           cp $HOME/.m2/settings.xml /tmp/settings.xml
           mv -vf .github/configs/settings.xml $HOME/.m2/settings.xml
 
+      - name: Install deps
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:1:1: Double quote to prevent globbing and word splitting

(shellcheck)